### ISSUE TITLE
Feature request: Playlist on hosted app

### ISSUE THREAD

**Issue by leocaseiro** [guancio/ChromeOsSubtitle#89]
Hi, thanks for this amazing app, that would be nice if we could build some playlists.

Good for series, tutorials and so on.

**Comment by vivekannan** [comment]
We do currently support playlists (kind of) Once you load multiple media
files, try right clicking anywhere on the screen and you will find a
playlist sub-menu. Is that what you are looking for?

**Comment by vivekannan** [comment]
Would also like to be a beta tester? I currently have a new version that I
haven't released for a long time because of lack of testing? If you are
interested, let me know.

**Comment by leocaseiro** [comment]
Hi @vivekannan. Thanks for your reply.

Btw, I meant next and previous navigation between the playlist.
Like, if I want to go to next chapter and so on...

PS: Yep! I would love to test it out.

**Comment by vivekannan** [comment]
Yes, you can do that as well. Ctrl + ] / [ for previous and next. Checkout
the wiki for the available shortcuts.

Let me get the new version ready and then i will send you a zip that you
can install into your browser.

**Comment by leocaseiro** [comment]
Wow, that's amazing. sorry about that. :D

Cheers, thanks!

**Comment by leocaseiro** [comment]
Oh! Just an update, I was actually mentioning about the web version:
https://subtitlevideoplayer.appspot.com/static/ChromeOsSubtitle/src/index.html

I'm not sure if you are the owner of that domain, but I want to use with Chromecast, thanks!

**Comment by vivekannan** [comment]
Yeah, I don't maintain the web version. Only the chrome app. The hosted one
still uses the old player. I will contact the owner and ask him if he is
cool with bumping the hosted app to the new versions. I will soon send you
the zipped version of the new app so that you can test, or you can just
grab it from the repo itself. Just install google-closure-linter and run
packaged.sh to get the zip.

**Comment by leocaseiro** [comment]
Hi @vivekannan, I managed to compile the app (I had to replace `cp --recursively` to `cp -R` because I'm on a Mac.

The new version is running nice. Thanks! The only drawback is that I can't chromecast.

Thanks for that!

If I get some free time, I'll try to implement chromecast for you guys. It shouldn't be too hard.

**Comment by vivekannan** [comment]
Yeah. Really need help with chromecast feature, had a lot of ideas for new
features (as seen in TODO file). Haven't done much though :(

**Comment by leocaseiro** [comment]
Hi! I was wondering how could I run locally as a webpage? Similar what it's running on the hosted app?

I've tried to run the `host./minimal_server.sh`. It loads, but I have so many 404 and js errors to debug...

I've tried run in node as well. But not lucky

**Comment by vivekannan** [comment]
Yeah, the source greatly depends on APIs provided only for registered apps/extensions and not for normal web pages. So I don't think the unmodified source can be directly run on a browser as a standard web page. :(

**Comment by leocaseiro** [comment]
I see, thanks @vivekannan
